README.md

```diff
@@ -427,18 +437,22 @@ If you use this package for binary classification or PU learning, please also ci
         publisher = {AUAI Press},
     }
 
-## Other Resources
+</details>
+
+## Other resources
 
-  - [Blogpost: Introduction to Confident Learning](https://l7.curtisnorthcutt.com/confident-learning)
+  - [Blog post: Introduction to Confident Learning](https://l7.curtisnorthcutt.com/confident-learning)
   - [NeurIPS 2021 paper: Pervasive Label Errors in Test Sets Destabilize Machine Learning Benchmarks](https://arxiv.org/abs/2103.14749)
+  - [Cleanlab Blog](https://cleanlab.ai/blog/)
+
+## Join our community
 
-## Join our Community
```

Comment:
Do we also want to add links to our social media pages?

Reply:
I already added a Twitter follow link at the top of readme, so to me just having Slack invite here is good enough. Will let you and @cgnorthcutt decide if you want to add more links.

Reply:
its fine. we can remove or add more links later. not a big deal.

Reply:
we don't  use twitter much btw, linkedin would be better. but going to merge as is and we can update this later